src/cli/serve/deprecated_config.js

```diff
@@ -0,0 +1,16 @@
+import { forOwn, has, noop } from 'lodash';
+
+// deprecated settings are still allowed, but will be removed at a later time. They
+// are checked for after the config object is prepared and known, so legacySettings
+// will have already been transformed.
+export const deprecatedSettings = new Map([
+  [['server', 'xsrf', 'token'], 'server.xsrf.token is deprecated. It is no longer used when providing xsrf protection.']
```

Comment:
It just means that these would warn:
```js
checkForDeprecatedConfig({ 'server.xsrf.token': 1 })
checkForDeprecatedConfig({ server: { xsrf: { token: 1 } } })
```

But these would not log a warning (all though they are valid ways to specify config)
```js
checkForDeprecatedConfig({ 'server.xsrf': { token: 1 } })
checkForDeprecatedConfig({ server: { 'xsrf.token': 1 } })
```